recipe/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -exo pipefail
+
+if [[ "${cuda_compiler_version:-None}" != "None" ]]; then
+    export CUTLASS_PATH=$(pwd)
+      if [[ ${cuda_compiler_version} == 12* ]]; then
+        export CUDA_HOME="${BUILD_PREFIX}/targets/x86_64-linux"
+        export CUDA_TOOLKIT_INCLUDE_DIR="${PREFIX}/targets/x86_64-linux"
+        export CUDA_INSTALL_PATH=$(which nvcc | awk -F'/bin/nvcc' '{print $1}')
+    fi
```

Comment:
there is a mismatch between the if and fi indentation

Reply:
Thanks, done in https://github.com/conda-forge/warp-lang-feedstock/pull/13/commits/45fa87c2148afe408423441bb6d6afec134ebf36